enhancements/kube-apiserver/audit-policy.md

```diff
@@ -164,7 +164,7 @@ are never logged beyond metadata level, and
 - `oauthaccesstokens.oauth.openshift.io`
 - `oauthauthorizetokens.oauth.openshift.io`.
 
-are not logged at all (because even their names are security-sensitive).
+are not logged at all (because even their names are security-sensitive). Oauthaccess tokens will be deleted to capture logouts.
```

Comment:
this is the paragraph assuming insecure tokens in the system. Your sentence must go below into the other section. There you can also update the policy rule.